pkgs/applications/blockchains/solana-validator/default.nix

```diff
@@ -56,8 +61,22 @@ rustPlatform.buildRustPackage rec {
     inherit hash;
   };
 
-  # partly inspired by https://github.com/obsidiansystems/solana-bridges/blob/develop/default.nix#L29
-  inherit cargoHash;
+  # fix build with rust 1.70+
```

Comment:
Wouldn't be using ``name = "rustc-1.7x.patch";`` better?

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-updates says a comment should accompany a patch. I like having a self-documenting name but I've been dinged by reviewers for missing a comment so usually just add both to expedite reviews but will accept either in changes i review.